### DISCUSSION TITLE
Maps functionality to discover

### DISCUSSION THREAD

**Discussion by s2b-git** [YetiForceCompany/YetiForceCRM#15046] [How to]
Installed in demo Map module is kind of limited in functionalities. How to learn more about it? I am interested in geocoding based distance calculations, pinpointing etc. Is it possible?

best

**Comment by TonyM15** [comment]
Interested too. Min package for map xtension - For ≤20 users is 20 Euro
How those users are calculated? I have 300 users but I want to allow maps for only 5 users with certain role. The rest wont get access to this module. So... how to resolve it?

**Reply by KatarzynaUlichnowska** [reply]
@TonyM15 
The prices of all our add-ons depend on the number of system users. The word “users” means employees who will be added to the YetiForce CRM system as users (System setting -> Permissions -> Users). In our opinion, this is the most optimal way to settle add-ons and we don’t plan to change it.
If in your case you have 300 system users, and only 5 of them will use YetiForce Map, we can offer you a discount. When you buy an add-on for a year, we can offer you a 20% discount, and if you buy it for 2 years, we can give you a 50% discount. Please let me know if such a proposal satisfies you.

**Comment by KatarzynaUlichnowska** [comment]
@s2b-git 
Unfortunately, we don’t understand what you mean by calculating the distance based on geocoding.
Do you mean situations when you can track the distances in the system that your sales representatives are currently traveling? Or maybe you mean determining the distance to a destination based on where your sales representatives are currently. Please provide more details about your ideas.
